Makefile

```diff
@@ -116,6 +116,7 @@ ci: globals check-env-vars ## Set Environment to CI
 	$(eval export NEW_ACCOUNT_EMAIL_ADDRESS=${ALERT_EMAIL_ADDRESS})
 	$(eval export ENV_SPECIFIC_CF_MANIFEST=cf-default.yml)
 	$(eval export ENABLE_DATADOG=true)
+	$(eval export DEPLOY_ENV ?= master)
```

Comment:
> Ci only has default name, as you could decide to operate on build instead of master.

Would you use `alphagov/paas-cf` against the build environment? Isn't it managed by `alphagov/paas-release-ci`?